test/components/popup.spec.tsx

```diff
@@ -98,28 +134,30 @@ describe('<Popup />', function() {
                 <span data-automation-id="SPAN">Popup Body</span>
             </Popup>);
 
+        const popup = new PopupTestDriver(result as Popup);
         return waitFor(() => {
-            expect(bodySelect(portalId)!.getBoundingClientRect().width)
+            expect((popup.root! as Element).getBoundingClientRect().width)
                 .to.equal(anchor.getBoundingClientRect().width);
         });
     });
 
     it('sets the default maxHeight', function() {
-        clientRenderer.render(
+        const {result} = clientRenderer.render(
             <Popup
                 anchor={anchor}
                 open={true}
             >
                 <span data-automation-id="SPAN">Popup Body</span>
             </Popup>);
 
+        const popup = new PopupTestDriver(result as Popup);
         return waitFor(() => {
-            expect(bodySelect<HTMLElement>(portalId)!.style.maxHeight).to.equal('500px');
+            expect((popup.root as HTMLElement).style.maxHeight).to.equal('500px');
```

Comment:
Technically, this is a problem with the portal driver but the popup root (Or the portal root) is a HTMLDivElement and there is no point in making it Element and then casting.
Maybe there is a simple solution in the portal driver for this?

Reply:
made an adjustment that makes root `HTMLElement` so all these castings are gone